BranchBrowser.py

```diff
@@ -754,15 +754,22 @@ def move_to_left(self):
                 self.submodules_left_listbox.insert(tk.END, selected_item)
 
 
+    def extract_feature_versions(self, team_name):
+        filtered_branches = [branch for branch in self.branches if branch.startswith(f"Features/{team_name}")]
+        feature_versions = list({branch.split("/")[-2] for branch in filtered_branches})
+        sorted_feature_versions = sorted(feature_versions, key=lambda v: float(v))
+        return sorted_feature_versions
+
+
     def update_repo_branches_right_listbox(self, event = None):
         # Get the selected values of the repo name and branch type comboboxes
         repo_name = self.repos_combobox.get()
         branch_type = self.branch_type_combobox.get()
 
-        if event == None or event.widget == self.repos_combobox:
+        if event and event.widget == self.repos_combobox:

```

Comment:
Ovde vidim da si pre izmena imao:

if event == None a sada ti stoji if event to su ti dva razlicita uslova, da li je nametno promenjeno? Ako nije, ako zelis da pitas jos uvek da li je If event == None, moras ici sa if not event

Reply:
Promenio sam logiku u medjuvremenu, tako da sam namerno i stavio da bude samo event u okviru svih provera u toj metodi